training_config/biattentive_classification_network.json

```diff
@@ -0,0 +1,72 @@
+{
+  "dataset_reader":{
+    "type": "sst_tokens",
+    "use_subtrees": true,
+    "granularity": "5-class"
+  },
+  "validation_dataset_reader":{
+    "type": "sst_tokens",
+    "use_subtrees": false,
+    "granularity": "5-class"
+  },
+  "train_data_path": "https://homes.cs.washington.edu/~nfliu/files/sst/train.txt",
```

Comment:
I just uploaded the data to s3, so you can give AI2 s3 links here instead, if you prefer: `https://s3-us-west-2.amazonaws.com/allennlp/datasets/sst/{train,dev,test}.txt`.